extension/package.json

```diff
@@ -35,7 +35,8 @@
     "@types/node": "^18.11.18",
     "@types/q": "^1.5.5",
     "jest": "^29.3.1",
-    "ts-jest": "^29.0.5",
-    "typescript": "^4.4.3"
+    "ts-jest": "^29.0.3",
```

Comment:
```suggestion
    "ts-jest": "^29.0.5",
```

Reply:
Is this a version that is locked in?

Reply:
No but I don't see why it should be downgraded